src/main/kotlin/com/beust/kobalt/plugin/application/ApplicationPlugin.kt

```diff
@@ -29,8 +29,8 @@ class ApplicationConfig {
 }
 
 @Directive
-fun Project.application(init: ApplicationConfig.() -> Unit) {
-    ApplicationConfig().let { config ->
+fun Project.application(init: ApplicationConfig.() -> Unit): ApplicationConfig {
```

Comment:
I'm not sure it's useful but I like that you introduced consistency to all these directives, so they all return the created config. I guess it doesn't hurt, and it's better than returning `Unit` anyway.